src/main_computer_urc/src/DriveTrainManager/main.cpp

```diff
@@ -1,51 +1,51 @@
-#include "rclcpp/rclcpp.hpp"
```

Comment:
we might want to consider setting up clang-format or some linter so we don't get all these formatting changes clogging up diffs.

Reply:
oh yeah, that's a good idea I didn't even know that's a thing. maybe we could give this to someone on software to look into?

Reply:
Yeah formatters are pretty nice for large projects but you end up needing a basic CI pipeline to check that the format rules are met